rllib/core/torch/torch_rl_module.py

```diff
@@ -0,0 +1,18 @@
+import torch
+
+from ray.rllib.core.rl_module import RLModule
+
+
+class TorchRLModule(torch.nn.Module, RLModule):
```

Comment:
This looks a bit different than the original design: https://github.com/ray-project/ray/pull/25682/files#diff-56205bdce1995828472f98db75622f44908b50611063270544c90c15756aefeb